src/components/AppLayout/Sidebar/SafeHeader/index.tsx

```diff
@@ -42,6 +42,11 @@ const StyledIcon = styled(Icon)`
   svg {
     height: 26px;
     width: 26px;
+    transform: rotateZ(-90deg);
+
+    path:nth-child(2) {
+      display: none;
+    }
```

Comment:
This is to remove the circle and make the arrow face right.

<img width="207" alt="Screenshot 2021-08-18 at 12 46 50" src="https://user-images.githubusercontent.com/381895/129885242-33b5ea08-7790-4251-8ecc-979b714a416f.png">